### ISSUE TITLE
Docker: Test ES volume and 03-full_stack

### ISSUE THREAD

**Issue by nollymar** [dotCMS/core#18445]
Recently, some changes were implemented to support docker secrets in our `03-full_stack` reference implementation. In addition to that, another issue was fixed: when containers were deleted, the volume that contained indexes weren't being preserved.

All these changes were included on this PR: https://github.com/dotCMS/docker/pull/25

Now, it is required to test these scenarios:

1. `docker-compose down` should not blow up indexes when using `01-demo`, `02_single-node` and `03-full_stack` implementations. In order to test it, run these commands on each reference implementation directory: 
a) `docker-compose up` to start services. Verify that a dotCMS instance is running and indexes were created correctly
b) `docker-compose down` to shutdown services and delete containers
c) `docker-compose up` again. Check if the same indexes in a) exist with green health

2. Run  `03-full_stack` reference implementation using `docker-compose up` and verify it is running, indexes should exist and an ES cluster with 3 master nodes should have been created. Then, try to scale the dotcms service running: `docker-compose up -d --scale dotcms=2`, verify that a new dotcms node was added to the dotCMS cluster

3. Test docker secrets on `03-full_stack`. The steps to follow are explained with more details on this task: https://github.com/dotCMS/core/issues/17843. Then, using this documentation https://docs.docker.com/engine/swarm/swarm-tutorial/scale-service/ scale the dotcms service to add a new dotcms node to the cluster

**Important note:** Tests 2 and 3 require to allocate at least 10GB of memory for Docker

**Comment by fabrizzio-dotCMS** [comment]
I followed the steps listed above.
**For 01-demo**
I started 01-demo with `docker-compose up` checked the index names, created some content then stopped the container via `docker-compose down` Once again I  re-started re-verified the index name hasn't changed and the content created in the first attempt was still there.

However, in the system configuration > network, The node info was gone after the restart of the container.

![Screen Shot 2020-05-07 at 16 17 33](https://user-images.githubusercontent.com/37148755/81372412-8b609900-90b7-11ea-8a28-aada2ee38c83.png)

**For 03-full_stack**

I started 03-full_stack with `docker-compose up` checked the index names, created some content then stopped the container via docker-compose down Once again I  re-started re-verified the index name hasn't changed and the content created in the first attempt was still there.

Then I ran `docker-compose up -d --scale dotcms=2`

I was able to verify the number of nodes was bumped up in 1. No restart was needed.

It is important to mention that a license.zip must be used for multiple nodes to work. Otherwise, the scale command will not take effect.  

finally: I rebuilt the docker-file using once again a remote branch without the db.properties file to force dotCMS picking up the docker secrets as it is described in  https://github.com/dotCMS/core/issues/17843

after building the new image using this command 
`docker build --pull --no-cache --build-arg BUILD_FROM=COMMIT --build-arg BUILD_ID=origin/issue-18445-docker-QA-branch -t dotcms-test .
`
I  ran `docker stack deploy -c docker-compose.yml dotcms_stack`
from docker/referenceImplementations/03-full_stack
The server once again started as expected. Then I finally ran 

`docker service ls` to get the service id followed by 

`docker service scale <SERVICE-ID>=<NUMBER-OF-TASKS>`

Where NUMBER-OF-TASKS. was replaced by 2.

Once again the running container was able to recognize the command I was able to verify that the number of nodes was this time 2.
